src/components/startup-form.tsx

```diff
@@ -0,0 +1,184 @@
+"use client"
+
+import { useForm } from "react-hook-form";
+import { zodResolver } from "@hookform/resolvers/zod";
+import { z } from "zod";
+import { Button } from "@/components/ui/button";
+import { Form, FormControl, FormField, FormItem, FormLabel, FormMessage } from "@/components/ui/form";
+import { Input } from "@/components/ui/input";
+import { database } from "@/app/firebaseConfig";
+import { ref as dbRef, get, push, set } from "firebase/database";
+import { toast } from "sonner";
+import { useRouter } from "next/navigation";
+import { useEffect } from "react";
+
+const formSchema = z.object({
```

Comment:
Could you move this to a folder in shared/api/validation 
there you create startupSchema.ts